src/api/app/models/package.rb

```diff
@@ -120,6 +116,23 @@ def self.check_access?(dbpkg = self)
     return Project.check_access?(dbpkg.project)
   end
 
+  def self.dirty_backend_packages
+    # avoid scope here, it will turn objects readonly
```

Comment:
I need the objects from the scope, just not read-only. So unscope would not help, but maybe there is another way to avoid the read-only flag?